ateam-common-packets/include/basic_control.h

```diff
@@ -0,0 +1,41 @@
+/**
+ * @file basic_control.h
+ * @author Matthew Barulic
+ * @brief definition for Basic Control data type
+ * @version 0.1
+ * 
+ * @copyright Copyright (c) 2022
+ *
+ */
+
+
+#pragma once
+
+#include <stdint.h>
+
+// if C11 or newer is used, include assert_static macro for testing
+#if defined(__cplusplus) || (defined( __STDC_VERSION__) && __STDC_VERSION__ >= 201112L)
+#include <assert.h>
+#endif
+
+typedef enum KickRequest {
```

Comment:
I think we should add a type here to clarifiy ready vs discahrage? Maybe KR_ARMED, KR_DISABLE? KR_ARM would keep the device charged, but no kick action would occur ever. KR_DISABLE would initiate a discharge.